lib/scholar/manifold/trimap.ex

```diff
@@ -188,13 +188,19 @@ defmodule Scholar.Manifold.Trimap do
     final_samples = Nx.broadcast(Nx.s64(0), shape)
 
     {final_samples, key, _, _} =
-      while {final_samples, key, rejects, i = Nx.s64(0)}, i < elem(shape, 0) do
-        {samples, key} = Nx.Random.randint(key, 0, opts[:maxval], shape: {elem(shape, 1)})
+      while {final_samples, key, rejects, i = Nx.u32(0)}, i < elem(shape, 0) do
+        # TODO: See if we can relax the samples to u32
```

Comment:
@msluszniak I kept the type here as s64 because if we change the type, we change the samples, which changes the result of the tests. Can you please tell me if you copied the tests from Python's trimap? Because if not, it is ok for us to change the values here and then update the tests. :)